images/linux/scripts/installers/dotnetcore-sdk.sh

```diff
@@ -32,8 +32,12 @@ for version in ${DOTNET_VERSIONS[@]}; do
     release_url="https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/${version}/releases.json"
     download_with_retries "${release_url}" "." "${version}.json"
     releases=$(cat "./${version}.json")
-    sdks=("${sdks[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdk.version'))
-    sdks=("${sdks[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdks[]?' | jq '.version'))
+    if [[ $version == "6.0" ]]; then
+        sdks=("${sdks[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdks[]?' | jq '.version' | grep -v preview | grep -v rc | grep -v display -m 1))
```

Comment:
Too many `greps` I believe 😃 Everything can be done using `jq` as it's a very powerful tool
```suggestion
        sdks=("${sdks[@]}" $(echo "${releases}" | jq -r 'first(.releases[].sdks[]?.version | select(contains("preview") or contains("rc") | not))'))
```

Reply:
Completely agree there are too many `jp` and `grep`, but in my guess your suggestion is harder for understanding and it has the same length though `or contains("display")` is missed 😃 The same approach is used [a bit lower in this file](https://github.com/actions/virtual-environments/blob/6865a4ccfe6c5bcf5ac3a96c4eb84d035136903c/images/linux/scripts/installers/dotnetcore-sdk.sh#L40) so it keeps the consistency.

Reply:
In my opinion, `jq` string is much more readable that's why I suggest reworking the other strings as well and getting rid of so many grep pipes
Why do we need this `display`?